x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/LoadMapping.java

```diff
@@ -82,7 +81,7 @@ private static void walkMapping(String name, Object value, Map<String, EsField>
             // extract field type
             DataType esDataType = getType(content);
             final Map<String, EsField> properties;
-            if (esDataType == OBJECT || esDataType == NESTED) {
+            if (esDataType == OBJECT) {
```

Comment:
This is from `Types` in QL, but it's only used in testing in ESQL. I think this safe for now.